Update to 1.20.4

TheMrEngMan opened the pull request:
Fixes #5 
Bump version to 1.2.0

Draft pull request as there might be a better way of doing this, I simply copied the code from `MathHelper`, as `multiplyColors` got removed in the 1.19.4 version.

Build in case anyone needs it:
https://github.com/TheMrEngMan/Colorful-Subtitles/releases/tag/1.2.0

TheMrEngMan committed:
Update to 1.19.4
Bump version to 1.2.0

BeeFox-sys commented:
This is also what i did to update to 1.19.4; i don't think of any other good way really

TheMrEngMan committed:
Updated to Minecraft 1.20.1
Changed to use ColorHelper class instead of copying the relevant method.
Bumped version to 1.3.0.

TheMrEngMan committed:
Updated to depend on Minecraft 1.20.1 only

TheMrEngMan commented:
Fixes #6 
Updated to Minecraft 1.20.1
Bump version to 1.3.0

Build in case anyone needs it:
https://github.com/TheMrEngMan/Colorful-Subtitles/releases/tag/1.3.0

TheMrEngMan committed:
Updated to Minecraft 1.20.4
Bumped version to 1.3.1

TheMrEngMan commented:
Updated to Minecraft 1.20.4
Bump version to 1.3.1

Build in case anyone needs it:
https://github.com/TheMrEngMan/Colorful-Subtitles/releases/tag/1.3.1
